README.md

```diff
@@ -58,6 +58,9 @@ Test geoips_clavrx installation
 
     # Ensure GeoIPS Python environment is enabled.
 
+    # Install the clavrx test data repo
+    git clone https://github.com/NRLMMD-GEOIPS/test_data_clavrx $GEOIPS_TESTDATA_DIR/
+
     # If you have all test data repos available:
     $GEOIPS_PACKAGES_DIR/geoips_clavrx/tests/test_all.sh
```

Comment:
Do you want to change this to just one of the tests/scripts commands - I think running test_all.sh is overkill.  Maybe the abi Cloud-Base-Height test script?

Reply:
It might be overkill, but it's something that we can make consistent between packages, right? It doesn't take very long as it is. I'd just leave it for now.